gammapy/image/models/tests/test_models.py

```diff
@@ -67,15 +68,31 @@ def test_sky_shell():
     assert_allclose(val.value, desired)
 
 
+def test_sky_diffuse_constant():
+    model = SkyDiffuseConstant(
+        value='42 sr-1'
+    )
+    lon = [1, 2] * u.deg
+    lat = 45 * u.deg
+    val = model(lon, lat)
+    assert val.unit == 'sr-1'
+    assert_allclose(val.value, 42)
+
+
 @requires_dependency('scipy')
 @requires_data('gammapy-extra')
-def test_sky_template():
+def test_sky_diffuse_map():
     filename = ('$GAMMAPY_EXTRA/datasets/catalogs/fermi/Extended_archive_v18'
-                '/Templates/HESSJ1841-055.fits')
-    template = SkyTemplate.read(filename)
-    lon = 26.7 * u.deg
-    lat = 0 * u.deg
-    actual = template(lon, lat)[0]
-    desired = 0.00017506744188722223 / u.deg ** 2
-    # desired = 1.1553735159851262 / u.deg ** 2
```

Comment:
So do you know why we cannot reproduce the old test value?

Reply:
I didn't have a look at the old values. I don't think it's worth, becase a) there were onlly 1-2 users of this (Axel + ?) and the behaviour wasn't well specified anyways. E.g. I think the old version was applying normalisation on init, and integrating over pixels, which are two things I think this class shouldn't do. Note that I left some TODO in the class of things I wanted to discuss with you concerning diffuse models. This is just a rough first draft. Do you have time today or later this week?